src/python/grpcio_testing/grpc_testing/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
nit: We might need to include Copyright here, otherwise it will fail on import to Google. Same for other files.

Reply:
Good point. I'll look into why the copyright check didn't catch this.

Reply:
I've put together https://github.com/grpc/grpc/pull/26353 to take care of this problem in general.

Reply:
We might need to sync with master after #26353 is merged.